### PR TITLE
Update office-365-u.s.-government-dod-endpoints.md

### LINKED ISSUE #12100: Include note for Cross Cloud Anonymous Join for the Teams endpoint section
Teams in DoD supports Cross-Cloud Anonymous Join, which means a DoD Teams user/client can join WW or GCCH hosted meetings. This has caused some confusion/lead to customers raising cases because they have whitelisted for DoD, but not for the other clouds. It is currently only implied that if they want to be able to connect to GCCH/WW meetings, that they have to whitelist for those endpoints.

Could we please include a warning/notification under the Teams subsection that for Cross-Cloud Join to properly work, they must whitelist the endpoints for the cloud of the target meeting? Also add links to the other cloud endpoint pages.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 13e07b8c-58ce-5d96-14cd-535b6d9c542c
* Version Independent ID: 13e07b8c-58ce-5d96-14cd-535b6d9c542c
* Content: [Office 365 US Government DOD endpoints - Microsoft 365 Enterprise](https://learn.microsoft.com/en-us/microsoft-365/enterprise/microsoft-365-u-s-government-dod-endpoints?view=o365-worldwide)
* Content Source: [microsoft-365/enterprise/microsoft-365-u-s-government-dod-endpoints.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/enterprise/microsoft-365-u-s-government-dod-endpoints.md)
* Service: **microsoft-365-enterprise**
* GitHub Login: @kelleyvice-msft
* Microsoft Alias: **kvice**

### PR DESCRIPTION
**Pull request by msbemba**
Added a note about cross cloud join

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12100

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/includes/office-365-u.s.-government-dod-endpoints.md [hunk 1]
```diff
@@ -1,35 +1,42 @@
...
 22 | Allow<BR>Required | Yes | `endpoint1-proddodcecompsvc-dodc.streaming.media.usgovcloudapi.net, endpoint1-proddodeacompsvc-dode.streaming.media.usgovcloudapi.net`<BR>`52.181.167.113/32, 52.182.52.226/32` | **TCP:** 443
-
-## Microsoft 365 Common and Office Online
+  
+Notes for this table:
+
+For  Cross-Cloud Join to properly work, you must whitelist the endpoints for the cloud of the target meeting
+  
+  [Office 365 URLs and IP address ranges](/microsoft-365/enterprise/urls-and-ip-address-ranges)
+  [Office 365 U.S. Government GCC High endpoints](/microsoft-365/enterprise/microsoft-365-u-s-government-gcc-high-endpoints)
```
Comment (JohanFreelancer9) [inline microsoft-365/includes/office-365-u.s.-government-dod-endpoints.md]:
```suggestion
> [!NOTE]
> For  Cross-Cloud Anonymous Join to work properly, you must add these endpoints for the cloud of the target meeting to the safe senders list:
> - [Office 365 URLs and IP address ranges](/microsoft-365/enterprise/urls-and-ip-address-ranges)
> - [Office 365 U.S. Government GCC High endpoints](/microsoft-365/enterprise/microsoft-365-u-s-government-gcc-high-endpoints)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kelleyvice-msft Please merge this PR. Thanks

**Pull request merged**